### ISSUE TITLE
Adding SSLR dataset

### ISSUE THREAD

**Issue by jtcramer** [soundata/soundata#94]
https://www.idiap.ch/en/dataset/sslr

**Comment by jtcramer** [comment]
Ahh actually the Zenodo page where the data is hosted indicates that you have to request access so this probably can't be added, right?

**Comment by magdalenafuentes** [comment]
Yes it can be added anyway! We have many of those in `mirdata`. What we do is to provide instructions in the download function indicating that you have to request access and do the download manually, but once you've done it all functionalities of soundata will be available
